mapeditor/mainwindow.cpp

```diff
@@ -77,22 +77,26 @@ void MainWindow::loadUserSettings()
 	QSettings s(Ui::teamName, Ui::appName);
 
 	auto size = s.value(mainWindowSizeSetting).toSize();
-	if (size.isValid())
+	if(size.isValid())
 	{
 		resize(size);
 	}
 	auto position = s.value(mainWindowPositionSetting).toPoint();
-	if (!position.isNull())
+	if(!position.isNull())
 	{
 		move(position);
 	}
+	lastSavingDir = s.value(lastDirectorySetting).toString();
+	if(lastSavingDir.isEmpty())
+		lastSavingDir = QString::fromStdString(VCMIDirs::get().userCachePath().make_preferred().string());
```

Comment:
```suggestion
		lastSavingDir = QString::fromStdString(VCMIDirs::get().userDataPath().make_preferred().string());
```
Shouldn't it DataPath?